util/gen_log_guide.ts

```diff
@@ -0,0 +1,503 @@
+        '30|2021-04-26T14:23:38.8440000-04:00|bd|Bio II|0.00|10FF0001|Tini Poutini|4000B262|Midgardsormr|00|10851737|51654||e34ec8d3a8db783fe34f152178775804',
+      ],
+    },
+  },
+  networkGauge: {
+    examples: {
+      en: [
+        '31|2019-11-27T23:22:40.6960000-05:00|10FF0001|FA753019|FD37|E9A55201|7F47|f17ea56b26ff020d1c0580207f6f4673',
+        '31|2021-04-28T00:26:19.1320000-04:00|10FF0002|BF000018|10035|40006600|00|f31bf7667388ce9b11bd5dd2626c7b99',
+      ],
+    },
+  },
+  ActorControl: {
+    regexes: {
+      network: NetRegexes.network6d({ capture: true }).source,
+      logLine: Regexes.network6d({ capture: true }).source,
+    },
+    examples: {
+      en: [
+        '33|2021-04-26T14:17:16.7440000-04:00|80034E5B|8000000C|16|E74|00|00|55e425f79b5cadd724dbc718b86f70b1',
+        '33|2021-04-26T14:17:31.6980000-04:00|80034E5B|8000000C|16|FFFFFFFF|00|00|b543f3c5c715e93d9de2aa65b8fe83ad',
+        '33|2021-04-26T14:18:39.0120000-04:00|80034E5B|40000007|00|01|00|00|7a2b827bbc7a58ecc0c5edbdf14a2c14',
+      ],
+    },
+  },
+  NameToggle: {
+    // @TODO: There's a NetRegexes but no Regexes for this one?
+    /* regexes: {
+      network: NetRegexes.nameToggle({ capture: true }).source,
+      logLine: Regexes.nameToggle({ capture: true }).source,
+    }, */
+    examples: {
+      en: [
+        '34|2021-04-26T14:19:48.0400000-04:00|4001C51C|Dragon\'s Head|4001C51C|Dragon\'s Head|00|a7248aab1da528bf94faf2f4b1728fc3',
+        '34|2021-04-26T14:22:19.1960000-04:00|4000B283|Selene|4000B283|Selene|01|734eef0f5b1b10810af8f7257d738c67',
+        '34|2021-04-26T14:25:06.0720000-04:00|4000B262|Midgardsormr|4000B262|Midgardsormr|00|e3482e01bb9e1c358c23a843b6c63556',
+        '34|2021-04-26T14:18:00.9980000-04:00|10FF0001|Tini Poutini|10FF0001|Tini Poutini|00|5b840ff5e79d908b3f9a3e806a64fd0b',
+      ],
+    },
+  },
+  Tether: {
+    regexes: {
+      network: NetRegexes.network6d({ capture: true }).source,
+      logLine: Regexes.network6d({ capture: true }).source,
+    },
+    examples: {
+      en: [
+        '35|2021-04-26T17:27:07.0310000-04:00|40003202|Articulated Bit|10FF0001|Tini Poutini|0000|0000|0001|10029769|000F|0000||ad71d456437e6792f68b19dbef9507d5',
+        '35|2021-04-27T22:36:58.1060000-04:00|10FF0001|Tini Poutini|4000943B|Bomb Boulder|0000|0000|0007|4000943B|000F|0000||a6adfcdf5dad0ef891deeade4d285eb2',
+        '35|2021-06-13T17:41:34.2230000-04:00|10FF0001|Tini Poutini|10FF0002|Potato Chippy|0000|0000|006E|1068E3EF|000F|0000||c022382c6803d1d6c1f84681b7d8db20',
+        '35|2021-04-26T21:12:22.3130000-04:00|4000158E|Copied Knave|40001590|Knave Of Hearts|0000|0000|0098|40001590|000F|0000||f980da434aac6cfb97c19ec26359c3b9',
+        '35|2021-04-26T19:10:04.7570000-04:00|40009B34||40009B33||A1FA|0000|009C|40009B33|000F|7FF0||467cec67b35e7de046e5ed9ce8986a54',
+      ],
+    },
+  },
+  limitBreak: {
```

Comment:
Why is `limitBreak` lowercase vs other types?

Reply:
The keys on this object line up with the keys on `logDefinitions`. Only the keys on `logDefinitions` that have a corresponding `NetRegexes` function are ProperCase.

Reply:
Hmm, ok, then `logDefinitions` is inconsistent.  That doesn't need to be fixed up in this PR then, but I think consistent capitalization would be better both here and there.

Reply:
Yeah, that was mentioned in #3377, they were previously all camelCase but I adjusted some of them there to match the expected format downstream in triggers to avoid having to update 3000-something triggers.

Reply:
Fixed in #3386.

Reply:
Okay. Do you want me to rebase after that's merged, or are you going to resolve conflicts as part of merging this one?

Reply:
I can rebase my patch if that's easier for you.  I wasn't trying to create extra work for you.  :)

Reply:
I'll go ahead and rebase this one after yours is merged, since yours is the simpler change.